js/NewsChannel.js

```diff
@@ -0,0 +1,128 @@
+  categoryElement.appendChild(catergoryText);
+  categoryDiv.appendChild(categoryElement);
+
+  let selectElement =document.createElement("select");
+  selectElement.setAttribute('class','dropdown');
+  selectElement.setAttribute('OnChange','ShowHideDiv(this)');
+  categoryDiv.appendChild(selectElement);
+
+  for(var index =0 ;index< newsChannels.length; index++){
+    var option= document.createElement("option");
+    option.value= index;
+    option.text= newsChannels[index];
+    selectElement.appendChild(option);
+  };
+  let subscribeDiv =document.createElement('div');
+  let subscribeText = document.createElement('text');
+  subscribeText.innerHTML="SUBSCIBE";
+  subscribeText.setAttribute('class','subscribe');
+  subscribeDiv.appendChild(subscribeText);
+
+  let emailElement = document.createElement('input');
+  emailElement.setAttribute('type', 'text');
+  emailElement.setAttribute('placeholder','Email Address');
+  emailElement.setAttribute('class','email-address');
+  emailElement.setAttribute('id','txtEmailAddress');
+  subscribeDiv.appendChild(emailElement);
+
+  let subscribebutton= document.createElement('input');
+  subscribebutton.setAttribute('type', 'button');
+  subscribebutton.setAttribute('value','Subscribe');
+  subscribebutton.setAttribute('class','btn-subscribe');
+  subscribebutton.setAttribute('id','btnSubscribe');
+  subscribebutton.setAttribute('OnClick','SaveEmail()');
+  subscribeDiv.appendChild(subscribebutton);
+
+  categoryDiv.appendChild(subscribeDiv);
+  rightPannel.appendChild(categoryDiv);
+}
+
+//Channels Information
+function LoadChannelInfo(index){
+  let newsContainer= document.createElement('div');
+  newsContainer.setAttribute('class','news-container');
+  newsContainer.setAttribute('id','Channel'+index);
+  leftPannel.appendChild(newsContainer);
+  let newsImage= document.createElement('div');
+  newsImage.setAttribute('class','news-image');
+  newsContainer.appendChild(newsImage);
+  let newsContent= document.createElement('div');
+  newsContent.setAttribute('class','news-content');
+  newsContainer.appendChild(newsContent);
+  let newsTitle= document.createElement('text');
+  newsTitle.innerHTML= "Title " + index;
+  newsTitle.setAttribute('class','news-header');
+  newsContent.appendChild(newsTitle);
+  let newsJournal= document.createElement('div');
+  newsJournal.innerHTML= " Posted on 29 June //Category:Category " + index;
+  newsJournal.setAttribute('class','news-journal');
+  newsContent.appendChild(newsJournal);
+  let newsInfo= document.createElement('p');
+  newsInfo.innerHTML= "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Quis ipsum suspendisse ultrices gravida. Risus commodo viverra maecenas accumsan lacus vel facilisis. Consectetur adispiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliquauis ipsum suspendisse ultrices gravidasus commodo.";
```

Comment:
Use template literals for long/ multi-line strings.

Reply:
Using template strings  are part of next task.